nixos/doc/manual/installation/installing-kexec.section.md

```diff
@@ -0,0 +1,69 @@
+# "Booting" into NixOS via kexec (PXE) {#sec-booting-via-kexec}
```

Comment:
> (PXE)

Maybe I missed something, but there's no PXE involved here, is there?

Reply:
yes, you're right.